roles/openshift_provisioning_aws_facts/README.md

```diff
@@ -0,0 +1,43 @@
+OpenShift Provisioning AWS Facts
+=========
+
+Provides common facts and defaults for other AWS provisioning roles.
+
+Requirements
+------------
+
+None
+
+Role Variables
+--------------
+
+| Name                                   | Default value                                       |                     |
+|----------------------------------------|-----------------------------------------------------|---------------------|
+| openshift_cluster_id                   | default                                             | Cluster ID (combination of openshift_cluster_id and openshift_env_id are expected to be unique for all OpenShift clusters) |
```

Comment:
Yes, it is meant to mirror the Ops concept of cluster id and env. It is still to be determined how we would handle it with current inventory, but it is something we will have to define through the docs.